corehq/apps/accounting/management/commands/migrate_wire_invoice_pdfs.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management import BaseCommand
+
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.accounting.models import InvoicePdf, WireBillingRecord
+from corehq.util.couch import IterDB
+
+
+class Command(BaseCommand):
+    help = ("Migrate InvoicePdfs to have is_wire field")
+
+    def handle(self, *args, **options):
+        invoice_ids = WireBillingRecord.objects.values_list('pdf_data_id', flat=True)
+        db = InvoicePdf.get_db()
+
+        with IterDB(db) as iter_db:
+            for doc in iter_docs(db, invoice_ids):
+                doc['is_wire'] = True
+                iter_db.save(doc)
```

Comment:
Not actionable, but FYI you can also do this as
```python
from corehq.util.couch import iter_update, DocUpdate

def update_invoice(doc):
    doc['is_wire'] = True
    return DocUpdate(doc)

result = iter_update(db, update_invoice, invoice_ids, verbose=True)
```
And that'll do things like retry if there's a error, skip documents that are unchanged, and give some info in the output (if `verbose == True`).